packages/runtime/runtime-definitions/src/summary.ts

```diff
@@ -81,10 +81,13 @@ export interface ISummarizerNode {
      * Calls the internal summarize function and handles internal state tracking.
      * If unchanged and fullTree is false, it will reuse previous summary subtree.
      * If an error is encountered and throwOnFailure is false, it will try to make
-     * a summary with a pointer to the previous summary + a blob of outstanding ops.
+     * a simple summary with a pointer to the previous summary + a blob of outstanding ops.
```

Comment:
Is "simple" the right way to describe it? I think intention is for it to be fast, so probably "fast" is a better word to use here?

Reply:
Oh, I see, by simple we mean both "fast" summary, but also "safe" summary (i.e. when data store fails to produce regular one). I wonder if it's better to have enum here to clearly articulate intentions, even though some of those intentions will have same implementation.

Reply:
I couldn't think of a good self-describing term for it.. definitely open to suggestions.
The distinction for that term should be that it is a pointer to previous summary + blob of outstanding ops.
Maybe "differential" summary?

Reply:
I like differential